pkg/cmd/root.go

```diff
@@ -21,9 +21,11 @@ import (
 // NewRoot create a new root command and sets the cliVersion to the passed variable
 // TODO: Add version support on the command
 func NewRoot() *cobra.Command {
-	return &cobra.Command{
-		Use:   "getenvoy",
-		Short: "getenvoy",
-		Long:  "getenvoy",
-	}
+	return rootCmd
+}
+
+var rootCmd = &cobra.Command{
```

Comment:
nit: Are these changes intentional or mis-changes from rebasing? I think this should be separately committed for the record and for the better reviews (OK for this time).

Reply:
These are intentional changes due to a rebase.